pipeline/ingestor/ingestor_common/src/main/scala/weco/pipeline/ingestor/common/models/AggregatableValues.scala

```diff
@@ -0,0 +1,114 @@
+package weco.pipeline.ingestor.common.models
+
+import io.circe.syntax._
+import io.circe.{Encoder, Json}
+import weco.catalogue.display_model.Implicits._
+import weco.catalogue.display_model.languages.DisplayLanguage
+import weco.catalogue.display_model.locations.DisplayLicense
+import weco.catalogue.display_model.work._
+import weco.catalogue.internal_model.identifiers.DataState
+import weco.catalogue.internal_model.work.{Availability, WorkData}
+
+import java.time.{LocalDateTime, ZoneId}
+
+/** We store aggregatable values in the Elasticsearch documents we store for the API.
+  *
+  * These values are serialised with the display models, so the API can read these values
+  * and drop them directly into an API response, without needing to know about what the
+  * display models look like.
+  *
+  * See https://github.com/wellcomecollection/docs/tree/main/rfcs/049-catalogue-api-aggregations-modelling
+  *
+  */
+trait AggregatableValues {
+  implicit class WorkDataOps(workData: WorkData[DataState.Identified]) {
+    def genreAggregatableValues: List[String] =
+      workData.genres
+        .map(DisplayGenre(_, includesIdentifiers = false))
+        .asJson(_.update("concepts", Json.fromValues(List())))
```

Comment:
This was a bug in the first version of this code – it would preserve the `concepts` field, so we'd (in theory) be aggregating over genres plus concepts, which isn't what we want.

The existing behaviour is to have an empty list of concepts in the genre aggregation; arguably it's not useful and we could look at changing it, but I don't want to be changing the external behaviour at the same time as internal refactoring.